protocol-gen/package.json

```diff
@@ -18,7 +18,7 @@
     "typescript": "^4.2.3"
   },
   "dependencies": {
-    "@dcl/protocol": "^1.0.0-3518293997.commit-bf08d5c",
+    "@dcl/protocol": "https://sdk-team-cdn.decentraland.org/@dcl/protocol/branch//dcl-protocol-1.0.0-3602487709.commit-3dbbf8f.tgz",
```

Comment:
TODO: change dependency after https://github.com/decentraland/protocol/pull/86 is merged